src/base/solvers/changeCobraSolver.m

```diff
@@ -266,8 +266,20 @@
 % Attempt to set the user provided solver for all optimization problem types
 if strcmpi(solverType, 'all')
     for i = 1:length(OPT_PROB_TYPES)
-        changeCobraSolver(solverName, OPT_PROB_TYPES{i}, 0);
-        fprintf([' > CBT_', OPT_PROB_TYPES{i}, '_SOLVER has been set to ', solverName, '.\n']);
+        try
```

Comment:
Good point.
Actually this way, a no existing solver would throw an error, while an existing one can display what it was set for.
I updated the pr accordingly.